src/test/java/net/openhft/chronicle/queue/impl/single/PretoucherTest.java

```diff
@@ -119,7 +119,7 @@ public void shouldHandleCycleRollByPretoucher() {
                 }
                 clock.addAndGet(50 + earlyMillis);
                 BackgroundResourceReleaser.releasePendingResources();
-                assertEquals(i + 2, capturedCycles.size());
+                assertEquals(i + 2.5, capturedCycles.size(), 0.5);
             });
 
             assertEquals(11, capturedCycles.size());
```

Comment:
This got changed to 12 in develop, but I had to take it back to 11, I assume it's something to do with EA